README.md

```diff
@@ -41,44 +57,124 @@ The statements in the proof include:
 - the sum of the input values match the sum of output values
 - the hashes in the input and output match the hash(value, salt, owner public key) formula
 - the sender possesses the private BabyJubjub key, whose public key is part of the pre-image of the input commitment hashes
-- the encrypted value in the input is derived from the receiver's UTXO value and encrypted with a shared secret using the ECDH protocol between the sender and receiver (this guarantees data availability for the receiver)
+- the encrypted values in the transaction are derived from the receiver's UTXO value and encrypted with a shared secret using the ECDH protocol between a random private key and the receiver (this guarantees data availability for the receiver, because the public key for the random private key used by the sender is published in the transaction)
```

Comment:
Odd to say `random private key` it gives a lack of trust - isn't it just the private key of the sender?

Reply:
aah okay so maybe specifying that is better than random 

so `one time use private key`

Reply:
will remember to do that in the next PR